src/deploy/process.rs

```diff
@@ -160,6 +160,15 @@ pub async fn process_deploy(args: DeployArgs) -> Result<()> {
                 );
                 println!("{} Creating candy machine", CANDY_EMOJI);
 
+                // We set cache items onChain values to false, in case they are set to true
+                // from a previous run. This ensures the config items are actually uploaded.
+                cache
+                    .items
+                    .0
+                    .iter_mut()
+                    .for_each(|(_, item)| item.on_chain = false);
+                cache.sync_file()?;
+
```

Comment:
Should we add this logic to the other arm of the `if` statement (after line 121)? That would be a sanity check that all items are `on-chain = false` when the candy machine address is empty.

Reply:
That's a good idea.

Reply:
Will do.